src/App.jsx

```diff
@@ -1,104 +1,29 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getCommentById(postId) {
+  return (
+    commentsFromServer.filter(comment => comment.postId === postId) || null
+  );
+}
```

Comment:
The `getCommentById` function should return an empty array instead of `null` when no comments are found for a post. This ensures that the `comments` property in the `posts` array is always an array, which simplifies handling in components that consume this data.